core-web/libs/template-builder/src/lib/components/template-builder/utils/gridstack-utils.ts

```diff
@@ -266,3 +266,15 @@ function simulateRowSpace(boxes: DotGridStackWidget[] = [], newBox?: DotGridStac
 
     return rowSpace;
 }
+
+/**
+ * Based on the container source, it returns the identifier that should be used as reference.
+ *
+ * @param dotContainer
+ * @returns string
+ */
+export function getContainerReference(dotContainer: DotContainer): string {
+    return dotContainer.source === CONTAINER_SOURCE.FILE
+        ? dotContainer.path
+        : dotContainer.identifier;
+}
```

Comment:
This is not gridstack, this is very dotcms.

Reply:
moved!